src/main/java/uk/gov/companieshouse/api/accounts/controller/FixedAssetsInvestmentsController.java

```diff
@@ -0,0 +1,173 @@
+import uk.gov.companieshouse.api.accounts.model.rest.SmallFull;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.fixedassetsinvestments.FixedAssetsInvestments;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.service.impl.FixedAssetsInvestmentsService;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.accounts.transaction.Transaction;
+import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
+import uk.gov.companieshouse.api.accounts.utility.ErrorMapper;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.companieshouse.api.accounts.CompanyAccountsApplication.APPLICATION_NAME_SPACE;
+
+@RestController
+@RequestMapping(value = "/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/notes/fixed-assets-investments", produces = MediaType.APPLICATION_JSON_VALUE)
+public class FixedAssetsInvestmentsController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAME_SPACE);
+    private static final String TRANSACTION_ID = "transaction_id";
+    private static final String COMPANY_ACCOUNT_ID = "company_account_id";
+    private static final String MESSAGE = "message";
+
+    @Autowired
+    private FixedAssetsInvestmentsService fixedAssetsInvestmentsService;
+
+    @Autowired
+    private ErrorMapper errorMapper;
+
+    @Autowired
+    private ApiResponseMapper apiResponseMapper;
+
+
+    @PostMapping
+    public ResponseEntity create(@Valid @RequestBody FixedAssetsInvestments fixedAssetsInvestments,
+                                 BindingResult bindingResult,
+                                 @PathVariable("companyAccountId") String companyAccountId,
+                                 HttpServletRequest request) {
+
+        if (bindingResult.hasErrors()) {
+            Errors errors = errorMapper.mapBindingResultErrorsToErrorModel(bindingResult);
+            return new ResponseEntity<>(errors, HttpStatus.BAD_REQUEST);
+        }
+
+        Transaction transaction =
+            (Transaction) request.getAttribute(AttributeName.TRANSACTION.getValue());
+
+
+        try {
+            ResponseObject<FixedAssetsInvestments> response = fixedAssetsInvestmentsService
+                    .create(fixedAssetsInvestments, transaction, companyAccountId, request);
+
+            return apiResponseMapper
+                .map(response.getStatus(), response.getData(), response.getErrors());
+
+        } catch (DataException ex) {
+            final Map<String, Object> debugMap = createDebugMap(companyAccountId, transaction,
+                "Failed to update fixedAssetsInvestments resource");
+            LOGGER.errorRequest(request, ex, debugMap);
```

Comment:
The logging in the controller has been updated across all the api controllers. Using a LogHelper. 

`LoggingHelper.logException(companyAccountId, transaction,
                "Failed to retrieve fixedAssetsInvestments resource", ex, request);`

These should be updated to match.

Reply:
Ok - will update to match..